Classes/Database/RteImagesDbHook.php

```diff
@@ -120,7 +120,7 @@ public function transform_db($value)
                     $absoluteUrl = trim($attribArray['src']);
                     // Make path absolute if it is relative and we have a site path which is not '/'
                     $pI = pathinfo($absoluteUrl);
-                    if ($sitePath && !$pI['scheme'] && GeneralUtility::isFirstPartOfStr($absoluteUrl, $sitePath)) {
+                    if ($sitePath && !($pI['scheme'] ?? null) && GeneralUtility::isFirstPartOfStr($absoluteUrl, $sitePath)) {
```

Comment:
Instead of `??` the check should be done with `isset()` - because this is what is happening: checking if the variable is set.

Reply:
Same applies to all following `??` checks.

Reply:
Ack

Reply:
Done

Reply:
While working on fixing PHPStan errors in the extension, I noticed that this part can be dropped altogether, since pathinfo() never returns an array containing 'scheme':

`Offset 'scheme' on array{dirname: string, basename: string, filename: string, extension?: string} in isset() does not exist.`

See: https://www.php.net/manual/en/function.pathinfo.php#refsect1-function.pathinfo-returnvalues

However, this will be tackled in an upcoming pull request.